<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eli's MIDI Piano Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbCancelPlay">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stops either a play or record operation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>731</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tabPlay">
     <attribute name="title">
      <string>Play</string>
     </attribute>
     <widget class="QPushButton" name="pbPlayFile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Play a File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePlayDirectory">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>/home/pi/midi/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPlayDirectory">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Directory of Files</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Directory for Session</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbChooseDirectory">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRecord">
     <attribute name="title">
      <string>Record</string>
     </attribute>
     <widget class="QPushButton" name="pbStartListening">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Listen</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leRecordPrefix">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>eliMidi</string>
      </property>
     </widget>
     <widget class="QListView" name="lvRecordingSession">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>541</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPlayRecording">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbDeleteRecording">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePlayDirectory_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>/home/pi/midi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbChooseDirectory_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>TreeView</string>
     </attribute>
     <widget class="QTreeView" name="tvRecordingSession">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>841</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>SlideShow</string>
     </attribute>
     <widget class="QLineEdit" name="lePlayDirectory_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>/home/pi/Pictures</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbChooseDirectory_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_PlaySlideshow">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>341</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Slideshow</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>681</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>pbCancelPlay</zorder>
   <zorder>lblStatus</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEli_s_MIDI_Companion">
    <property name="title">
     <string>Eli's MIDI Companion</string>
    </property>
   </widget>
   <addaction name="menuEli_s_MIDI_Companion"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
